misc/graylog.conf

```diff
@@ -324,7 +324,7 @@ stream_aware_field_types=false
 # Multiple rotation strategies are supported, the default being "time-size-optimizing":
 #   - "time-size-optimizing" tries to rotate daily, while focussing on optimal sized shards.
 #      The global default values can be configured with
-#       "time_size_optimizing_rotation_min_lifetime" and "time_size_optimizing_rotation_max_lifetime".
+#       "time_size_optimizing_retention_min_lifetime" and "time_size_optimizing_retention_max_lifetime".
```

Comment:
ah, I forgot that when Sebastian asked me to change the naming.

Can we fix this for the variables and the getters as well? 
e.g. `timeSizeOptimizingRotationMinLifeTime` to `timeSizeOptimizingRetentionMinLifeTime`